File: BookStoreApi/BookStoreApi/Dtos/FileDto.cs
namespace BookStoreApi.Dtos;

/// <summary>
/// Dto файла
/// </summary>
public class FileDto
{
    /// <summary>
    /// Гуид
    /// </summary>
    public Guid Guid { get; set; }

    /// <summary>
    /// Байты
    /// </summary>
    public IEnumerable<byte> Bytes { get; set; }
}

File: frontend/src/app/dtos/file.dto.ts
/** Dto для файлов. */
export class FileDto {
    /** Гуид файла */
    public guid: string;

    /** Байты файла. */
    public fileBytes: Blob;

    constructor(guid: string, fileBytes: Blob) {
        this.guid = guid;
        this.fileBytes = fileBytes;
    }
}
